build-system/app.js

```diff
@@ -284,7 +284,23 @@ app.use('/form/redirect-to/post', (req, res) => {
 app.use('/form/echo-json/post', (req, res) => {
   cors.assertCors(req, res, ['POST']);
   const form = new formidable.IncomingForm();
-  form.parse(req, (err, fields) => {
+  const fields = Object.create(null);
+  form.on('field', function(name, value) {
+    if (name in fields) {
```

Comment:
I would say revert this condition and short-circuit to save yourself an indent level in the more complicated logic.

```js
if (!(name in fields)) {
  fields[name] = value;
  return;
}
// ...
```